schema/README.md

```diff
@@ -14,3 +14,15 @@ If a version is not specified the script will default to the latest published ve
 ```bash
 ACCEPT_EULA=<yes/no> ./schema/generate.sh
 ```
+
+## Adding Allowed Arguments for "Enumtypes"
+
+This section if for you who are developing and want to extend the set of allowed arguments for a property a given property.
+Let's take an example working with `HyperCubeDef`. This definition has a property called `qMode` which is represented by an enum called `NxHypercubeMode`.
```

Comment:
Link to https://core.qlik.com/services/qix-engine/apis/qix/definitions/#hypercubedef ?

We should also consider adding an enum page to the core.qlik.com docs.

Reply:
Sounds good to me!